zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -32,10 +34,26 @@
       return getContactAdminForm().find('#contact-admin-send-button');
     }
 
+    // Returns a span.
+    function getContactAdminMessageErrorField(){
+      return getContactAdminForm().find('#contactAdminForm\\:messageField\\:input\\:contact-admin-error-message\\:contactAdminForm\\:messageField\\:input\\:contact-admin-message');
+    }
+
     function toggleSendButton() {
       var hasMessage = getContactAdminMessageField().val() !== '';
       getContactAdminSendButton().prop('disabled', !hasMessage);
     }
+
+    function completeSending() {
+      var errMsg = getContactAdminMessageErrorField().text();
+      if (errMsg != '') {
```

Comment:
I tried to break it by this way
getContactAdminForm().find('#contactAdminForm\\:messageField\\:input\\:contact-admin-error-message\\:contactAdminForm\\:messageField\\:input\\:contact-admin-message').**text('');**

It is broken now and validation error is disable. However, the user cannot submit a looong message in any way because sending process is happening in the back end side. 

Please correct me if I am wrong or try to break it. I am not a good hacker.